src/actions/normalize.js

```diff
@@ -14,22 +13,20 @@ const PLURAL = "PLURAL";
  * Normalize YAML to localicious internal representation.
  *
  * @param {*} data The YAML data to normalize.
- * @param {*} platforms The platforms to extract.
+ * @param {*} languages The languages to be included.
+ * @param {*} collections The collections to extract.
+
  */
-const normalizeYaml = (data, platforms, languages) => {
-  return flatten(
-    platforms
-      .map(platform => aggregate(data[platform], platform, languages))
-      .concat(aggregate(data[platformKeywords.SHARED], platformKeywords.SHARED, languages))
-  );
+const normalizeYaml = (data, languages, collections) => {
+  return flatten(collections.map(collection => aggregate(data[collection], languages)));
```

Comment:
Do you know if we are doing any validation if a key is duplicated? or in case we don't are we adding both keys or are we replacing them?

Reply:
Currently, we overwrite the key if it's defined in multiple collections I guess. Would be nice to decide how we want to deal with duplications. I think we can either:
1) Merge the paths, but we'll still risk replacing parts of the branches
2) Report duplicated keys and fail the validation step

I personally think we should go for (2), I don't see a use case of duplicating keys over multiple collections and it will probably lead to unpredictable behavior.

Reply:
Agree, I also prefer option number 2

Reply:
Yep, agree that 2 is the way to go. Can you file an issue, such that we keep it on our radar?